bench/irmin-pack/tree.ml

```diff
@@ -761,9 +1193,19 @@ let results_dir =
   in
   Arg.(value @@ opt string default_results_dir doc)
 
+let setup_log style_renderer level =
```

Comment:
move this in common? as it is shared with layers.ml; 
but I'm not sure you should add this as we use `Printf` everywhere; if we do want to use Logs, we should switch to it everywhere in this file, no?